scripts/moonshine/v2/test.py

```
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+# Copyright      2026  Xiaomi Corp.        (authors: Fangjun Kuang)
+
+
+import math
```

Flake8 (7.3.0)

[error] 'math' imported but unused

(F401)